README.md

```diff
@@ -4,65 +4,85 @@
 
 # Disclaimer
 
-This project is currently on an experimental stage. We periodically synchronize the source code and test cases of this
-repository with the [CLI folder](https://github.com/apache/incubator-openwhisk/tree/master/tools/cli/go-whisk-cli) and the [test folder](https://github.com/apache/incubator-openwhisk/tree/master/tests) in OpenWhisk. The framework of test cases is under construction
-for this repository. Please contribute to the [CLI folder](https://github.com/apache/incubator-openwhisk/tree/master/tools/cli/go-whisk-cli) in OpenWhisk for any CLI changes, before we officially announce the separation
+This project is currently on an experimental stage. We periodically synchronize
+the source code and test cases of this repository with the [CLI
+folder](https://github.com/apache/incubator-openwhisk/tree/master/tools/cli/go-whisk-cli)
+and the [test
+folder](https://github.com/apache/incubator-openwhisk/tree/master/tests) in
+OpenWhisk. The framework of test cases is under construction for this
+repository. Please contribute to the [CLI
+folder](https://github.com/apache/incubator-openwhisk/tree/master/tools/cli/go-whisk-cli)
+in OpenWhisk for any CLI changes, before we officially announce the separation
 of OpenWhisk CLI from OpenWhisk.
 
 
-The OpenWhisk Command Line Interface (OpenWhisk CLI) is a unified tool that provides a consistent interface to
-interact with OpenWhisk services. With this tool to download and configure, you are able to manage OpenWhisk services
-from the command line and automate them through scripts.
-
+The OpenWhisk Command Line Interface (OpenWhisk CLI) is a unified tool that
+provides a consistent interface to interact with OpenWhisk services. With this
+tool to download and configure, you are able to manage OpenWhisk services from
+the command line and automate them through scripts.
 
 # Where to download the binary of OpenWhisk CLI
 
-The OpenWhisk CLI is available on the release page: [click here to download](https://github.com/apache/incubator-openwhisk-cli/releases).
-We currently have binaries available for Linux, Mac OS and windows under amd64 architecture. You can download the
-binary, which fits your local environment.
+The OpenWhisk CLI is available on the release page: [click here to
+download](https://github.com/apache/incubator-openwhisk-cli/releases). We
+currently have binaries available for Linux, Mac OS and windows under amd64
+architecture. You can download the binary, which fits your local environment.
 
 
 # How to build the binary locally
 
-You can also choose to build the binaries locally from the source code with Go tool.
+The OpenWhisk CLI is written in the Go language.  You have two options to build
```

Comment:
Remove double space before second sentence for consistency.

Reply:
Oh, c'mon.  It doesn't show when it renders!  Done anyway :)